File: packages/sdui-common/src/main/java/nl/q42/common/components/SpacerComponent.java
package nl.q42.common.components;

import nl.q42.common.VersionDependable;

@VersionDependable(until = 2)
public class SpacerComponent extends Component {
    public final int size;

    public SpacerComponent(int size, String contentId) {
        super(ComponentTypes.SPACER, contentId);
        this.size = size;
    }
}

File: packages/sdui-common/src/commonMain/kotlin/nl/q42/common/components/SpacerComponent.kt
package nl.q42.common.components

import kotlinx.serialization.*
import nl.q42.common.VersionDependable

@VersionDependable(until = 2)
@Serializable
@SerialName(ComponentTypes.SPACER)
class SpacerComponent(
    val size: Int,
    override val contentId: String
) :
    Component(ComponentTypes.SPACER)
